scripts/dependabot_helper.bash

```diff
@@ -26,6 +26,11 @@ for pr in "${prs[@]}"; do
             git push
             sleep 5
             gh pr review "${pr}" --approve --body "${body}"
+        else
+            status=$(gh pr view --json reviewDecision | jq '.reviewDecision')
+            if [[ "$status" == '"REVIEW_REQUIRED"' ]]; then
+                gh pr review "${pr}" --approve --body "${body}"
```

Comment:
looks like some go.mod changes in KEB do not require running `go mod tidy` under tools/kcp, but the script should still approve if PR requires approval